blivetgui/osinstall.py

```diff
@@ -198,6 +200,24 @@ def enlightbox(self):
 
         self.main_window.lightbox_off()
 
+    def _reraise_exception(self, exception, traceback, message):
+        allow_report = True
+        allow_ignore = self.allow_ignore and issubclass(type(exception), StorageError)
+        if allow_ignore:
+            msg = _("{message}\n{error}\n Please click Report button to raise the error and let anaconda \n to handle the report process if you want to report this.").format(message=message, error=str(exception))
+        else:
+            msg = _("Unknown error occured. Anaconda will be terminated.\n{error}").format(error=str(exception))
+
+        dialog = message_dialogs.ExceptionDialog(self.main_window, allow_ignore,
+                                                 allow_report, msg, str(traceback))
+        response = dialog.run()
+
+        if response == constants.DialogResponseType.BACK:
+            return
+        else:
+            if response == constants.DialogResponseType.REPORT:
+                raise type(exception)(message + str(exception) + "\n" + traceback)
+
```

Comment:
I didn't realize this before, but we need to handle the QUIT response here too. Right now it's basically same as BACK. Quit option in python-meh just calls `sys.exit` -- https://github.com/rhinstaller/python-meh/blob/master/meh/handler.py#L184 -- so I guess blivet-gui should do the same.